docs/walkthrough.md

```diff
@@ -106,15 +106,17 @@ provides. We can view the `ClusterServiceClass` resources available:
 
 ```console
 $ svcat get classes
-                NAME                        DESCRIPTION                         UUID
-+-----------------------------------+-------------------------+--------------------------------------+
-  user-provided-service               A user provided service   4f6e6cf6-ffdd-425f-a2c7-3c9258ad2468
-  user-provided-service-single-plan   A user provided service   5f6e6cf6-ffdd-425f-a2c7-3c9258ad2468
+                 NAME                  NAMESPACE         DESCRIPTION
++------------------------------------+-----------+-------------------------+
+  user-provided-service                            A user provided service
+  user-provided-service-single-plan                A user provided service
+  user-provided-service-with-schemas               A user provided service
 
 $ kubectl get clusterserviceclasses -o=custom-columns=NAME:.metadata.name,EXTERNAL\ NAME:.spec.externalName
```

Comment:
There's no longer a need to specify the custom output format. You should now be able to simply run `kubectl get clusterserviceclasses` and get the same nice output.

Reply:
Ok, thanks, fixed.